packages/sui-mono/src/config.js

```diff
@@ -35,6 +36,15 @@ module.exports = {
   },
   getPublishAccess: function () {
     return publishAccess
+  },
+  getProjectName: function () {
+    return projectPackage.name
+  },
+  isMonoPackage: function () {
+    const folders = cwds(path.join(basePath, packagesFolder), deepLevel)
+    const scopePath = folders.pop()
+
```

Comment:
I would say that `If one does not have the package.json is a multi-package`

What do you think ?!